setup.py

```diff
@@ -18,6 +18,7 @@
     "chunkify",
     "treeherder-client",
     "PGPy",
+    "buildtools",
```

Comment:
This is jus to make the tox tests happier. On the server we have them already installed (release-runner.py is there).